src/evaluations/tests/evaluator_test.py

```diff
@@ -178,7 +177,7 @@ def test_evaluate_all_save_results(self):
               f'{sketch_estimator_config.name}/{scenario_config.name}')
 
   def test_same_random_state_of_same_scenario(self):
-    test_evaluator = self.get_test_evaluator(self.create_tempdir().full_path)
+    test_evaluator = self.get_test_evaluator(self.create_tempdir())
```

Comment:
Why was this changed? This change breaks causes TypeErrors on my machine.

Reply:
I must have missed this when copying things over manually: my apologies.